common/components/ui/Modal.jsx

```diff
@@ -18,8 +18,7 @@ type Props = {
       | 'danger'
       | 'link',
     disabled?: boolean,
-    // $FlowFixMe - Why the fuck doesn't this like onClick?
-    onClick: () => void
+    onClick?: () => void
```

Comment:
Definitely aware, and not something I wanted those buttons to have (A button in a modal should always do something.) I was passing onClick, but it was still complaining, so I haven't quite figured that one out yet.